pkgs/development/python-modules/file-read-backwards/default.nix

```diff
@@ -0,0 +1,21 @@
+{ lib, buildPythonPackage, fetchPypi, mock }:
+
+buildPythonPackage rec {
+  pname = "file-read-backwards";
+  version = "2.0.0";
+
+  src = fetchPypi {
+    pname = builtins.replaceStrings ["-"] ["_"] pname;
```

Comment:
Use lib instead of builtins.

Reply:
Why?

Reply:
I remember someone saying something about lib enabling reflection, but I can't find it any more. @SuperSandro2000 can you point me to any documentation / discussion about why lib is preferred over builtins here?

Reply:
There is no documentation for every little thing. lib sometimes has some extra sugar on top of the builtins like fetchurl and it is just easier to use lib everywhere because you don't need to think about anything. For replaceStrings lib and builtins are identical.